### PR TITLE
save abi 

### PR DESCRIPTION
**Pull request by Sab94**

### CODE REVIEW

File: lib/components/ContractCompiled/index.js [hunk 1]
```diff
@@ -23,17 +23,21 @@ import InputsForm from '../InputsForm';
...
 
 class ContractCompiled extends React.Component {
     constructor(props) {
         super(props);
         this.helpers = props.helpers;
         this.state = {
             estimatedGas: 9000000,
-            ContractABI: props.interfaces[props.contractName].interface
+            ContractABI: props.interfaces[props.contractName].interface,
+            savePath: 'filename.abi'
```
Comment (0mkara) [inline lib/components/ContractCompiled/index.js]: `savePath` can be `${contractName}.abi`

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@
 npm-debug.log
 node_modules
 package-lock.json
+.idea/
```
Comment (0mkara) [inline .gitignore]: If not used please remove `.idea`

File: styles/atom-solidity.less [hunk 3]
```diff
@@ -392,3 +392,13 @@
 .test-header {
     text-align: center;
 }
+
+button.btn.icon.icon-desktop-download.inline-block-tight.icon-button {
```
Comment (0mkara) [inline styles/atom-solidity.less]: we don't need this whole thing to style the button. I guess something like `.icon .icon-desktop-download` should be good enough.

File: .gitignore (hunk 2 above)
Comment (Sab94) [reply .gitignore]: `.idea` contains the project specific settings for project for Intellij IDE & I don't what to add it in the project.

**Pull request merged**